doc/jsonrpc.md

```diff
@@ -9998,6 +9999,90 @@ Example response:
 ]
 ~~~
 
+### bdev_lvol_shallow_copy {#rpc_bdev_lvol_shallow_copy}
+
+Make a shallow copy of lvol over a given bdev. Only cluster allocated to the lvol will be written on the bdev.
+Must have:
+* lvol read only
+* lvol size smaller than bdev size
+* lvstore block size a multiple of bdev size
+
+#### Parameters
+
+Name                    | Optional | Type        | Description
+----------------------- | -------- | ----------- | -----------
+src_lvol_name           | Required | string      | UUID or alias of lvol to create a copy from
+dst_bdev_name           | Required | string      | Name of the bdev that acts as destination for the copy
+
+#### Example
+
+Example request:
+
+~~~json
+{
+  "jsonrpc": "2.0",
+  "method": "bdev_lvol_shallow_copy",
+  "id": 1,
+  "params": {
+    "src_lvol_name": "8a47421a-20cf-444f-845c-d97ad0b0bd8e",
+    "dst_bdev_name": "Nvme1n1"
+  }
+}
+~~~
+
+Example response:
+
+~~~json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": true
+}
+~~~
+
+### bdev_lvol_shallow_copy_status {#rpc_bdev_lvol_shallow_copy_status}
+
+Get shallow copy status
+
+#### Result
+
+This RPC reports if a shallow copy is still in progress and operation's advance state in the format
+_actual_cluster/total_clusters_to_copy_
+
+#### Parameters
+
+Name                    | Optional | Type        | Description
+----------------------- | -------- | ----------- | -----------
+src_lvol_name           | Required | string      | UUID or alias of source lvol
+
+#### Example
+
+Example request:
+
+~~~json
+{
+  "jsonrpc": "2.0",
+  "method": "bdev_lvol_shallow_copy_status",
+  "id": 1,
+  "params": {
+    "src_lvol_name": "8a47421a-20cf-444f-845c-d97ad0b0bd8e"
+  }
+}
+~~~
+
+Example response:
+
+~~~json
+{
+  "jsonrpc": "2.0",
+  "id": 1,
+  "result": {
+    "in_progress": true,
+    "status": "2/4"
+  }
+}
```

Comment:
Suggest like this
 
```
progress: "2/4" (or in percentage)
state: value (..."in progress", "complete", "error")
error: "error message"
```

Reply:
This is the sync version of the command, so the result is given when the call returns. Discussing with @shuo-wu we talked about an async call, and in that case I should add the result of the last operation to the returned values of the status call. But this can be weird because we have no way to identify an operation, we can only store the result of the last one.

Reply:
I have updated the description of the command:
`This RPC reports if a shallow copy is still in progress and operation's advance state in the format number_of_copied_clusters/total_clusters_to_copy`.